Find redis-server binary in other locations than /usr/bin

### What action do you want to perform

Use `shutil.which` to detect the location of the `redis-server` binary instead of using the currently hard-coded "/usr/bin/redis-server".

Maybe with a fallback to "/usr/bin/redis-server" if `which` doesn't find anything.

Something like:

    [...]
    from shutil import which
    [...]
    parser.addini(
        name="redis_exec",
        help=_help_exec, 
        default=which("redis-server") or "/usr/bin/redis-server"
    )

### What are the results


### What are the expected results

Tried to provide redis using `docker run -d redis`, with no joy, and then found this issue.

This could resolve an exception like:

```
    def _check_version(self):
        """Check redises version if it's compatible."""
        with os.popen(f"{self.executable} --version") as version_output:
            version_string = version_output.read()
        if not version_string:
            raise RedisMisconfigured(
>               f"Bad path to redis_exec is given:"
                f" {self.executable} not exists or wrong program"
            )
E           pytest_redis.executor.RedisMisconfigured: Bad path to redis_exec is given: /usr/bin/redis-server not exists or wrong program

/opt/conda/envs/aio-aws/lib/python3.7/site-packages/pytest_redis/executor.py:228: RedisMisconfigured
```

--- a/README.rst
+++ b/README.rst
@@ -155,7 +155,7 @@ You can pick which you prefer, but remember that these settings are handled in t
      - --redis-exec
      - redis_exec
      - -
-     - /usr/bin/redis-server
+     - Look in PATH for redis-server via shutil.which
    * - host
      - host
      - --redis-host

--- a/src/pytest_redis/plugin.py
+++ b/src/pytest_redis/plugin.py
@@ -16,6 +16,7 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with pytest-redis.  If not, see <http://www.gnu.org/licenses/>.
 """Plugin configuration module for pytest-redis."""
+from shutil import which
 from tempfile import gettempdir
 
 from pytest_redis import factories
@@ -42,7 +43,7 @@ _help_datadir = "Directory where test Redis instance data files will be stored"
 
 def pytest_addoption(parser):
     """Define configuration options."""
-    parser.addini(name="redis_exec", help=_help_exec, default="/usr/bin/redis-server")
+    parser.addini(name="redis_exec", help=_help_exec, default=which("redis-server"))
     parser.addini(name="redis_host", help=_help_host, default="127.0.0.1")
     parser.addini(
         name="redis_port",

